apps/web/pages/event-types/[type]/index.tsx

```diff
@@ -148,6 +149,17 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
     endDate: new Date(eventType.periodEndDate || Date.now()),
   });
 
+  const convertMinimumBookingNoticeToMinutes = (type: string, minNotice: number) => {
```

Comment:
I think we can move this to a similar util file in `packages/lib` as the one I mentioned above.